src/store/trees/trees.js

```diff
@@ -387,6 +393,27 @@ export default {
   },
 
   actions: {
+    async validateAndAddOrgLanguage({dispatch, commit}, {persistRoute, language}) {
+      const validationErrors = await dispatch('validation/validate_new_language', {language}, {root: true})
+      if (!validationErrors.isValid) {
+        return false
+      }
+
+      if (!persistRoute) {
+        console.info('Language persistence route not configured correctly in builder.config.json. Falling back to vuex store')
```

Comment:
As this event means we're having wrong configuration, I'd suggest we use `console.warn` instead. The same for the try catch below.

Reply:
I think when using the standalone builder it should be fine to not configure this route but it's still useful info to know you can if you want. I think I'm going to leave this as is.

Reply:
ok, thanks for that extra clarification